### DISCUSSION TITLE
Homekit router

### DISCUSSION THREAD

**Discussion by toshibochan** [RavenSystem/esp-homekit-devices#1535] [General]
I’m thinking to add homekit router in my house. Anyone have homekit router? I want know if have any issues or if everything is working.

**Comment by tonysprenk** [comment]
I’m using eero pro which is HomeKit enabled. It’s nice to be able to allow or disallow devices to be connected to the internet but that is basically the only feature, and most ideeën routers allow you to do that anyway. So my advice: don’t buy a router because it’s HomeKit compatible, but it because it’s the right router for you and if it has HomeKit than that’s nice.

**Comment by RavenSystem** [comment]
I personally don't recommend HomeKit routers. To be HomeKit, it accesses to all sensible and secure data from all HomeKit accessories. Those sensible information can not be shared, but there is not any mechanism to avoid it, leaving it in the trust of the router brand.

And there is not any real benefit. A Homekit router can block Internet access of Homekit accessories. Any decent router can block Internet access of ANY device. And seeing router into Home app doesn't make sense for me.

**Reply by toshibochan** [reply]
@RavenSystem when homekit router is set to Auto you still can update the esp-homekit-devices?

**Reply by RavenSystem** [reply]
Sorry, I don't know.

**Reply by toshibochan** [reply]
I did turn on and I was able to update the devices

**Comment by toshibochan** [comment]
I think homekit router block device to talk each other to.

**Comment by toshibochan** [comment]
Eero 6 with FW-6.4.0 not allowing me to activate homekit router. Anyone having this issue?

**Comment by toshibochan** [comment]
to anyone have eero routers....
I was having problem to activate HomeKit router in the eero. I contact the support to help me. I exchange couples email. and yesterday he was asking me my device Mac address, I give to him, I think was strange because why he want Mac address. today my router reboot 2 time and right after I received email telling me he change my router setting and to try activate HomeKit router. I was shocked he can access my router and see and make change in the settings. and he didn't ask me for permission to do so, and no system to block to any support personal to get access to any router all over the word. this router going back.

**Reply by RavenSystem** [reply]
As I said, HomeKit routers are not a good option, and Eero (Amazon) is not different.

**Reply by toshibochan** [reply]
The problem is they can access the router anytime they want

**Reply by toshibochan** [reply]
They can see and make settings changes and get all informations
